tests/integration_test_data_endpoint.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+import re
+
+from syncano.models import Class, DataEndpoint, Object, ResponseTemplate
+from tests.integration_test import InstanceMixin, IntegrationTest
+
+
+class DataEndpointTest(InstanceMixin, IntegrationTest):
+
+    schema = [
+        {
+            'name': 'title',
+            'type': 'string',
+            'order_index': True,
+            'filter_index': True
+
+        }
+    ]
+
+    template_content = '''
```

Comment:
@opalczynski I cannot rely on `html_table_list`. Tests should as independent as possible. Imagine that some core developer changes `html_table_list` format. Test would fail. With my approach tests are the most independent.